build-config/scripts/onie-mk-bin.sh

```diff
@@ -31,7 +31,7 @@ uimage_max_size=$(( 4 * 1024 * 1024 ))
 
 # Most platforms have a max u-boot size of 512KB.  You can override
 # this in the platform specific conf file.
-uboot_max_size=$(( 512 * 1024 ))
+uboot_max_size=$(( 768 * 1024 ))
```

Comment:
This is a common file used by all platforms.  This value can be overridden on a per platform basis by setting uboot_max_size in machine/fsl/fsl_p2041rdb/onie-rom.conf.